Utilities/bootstrap

```diff
@@ -652,10 +674,10 @@ def process_runtime_libraries(build, args, lib_path):
     runtime_swiftdoc_path = os.path.join(
         lib_path, "PackageDescription.swiftdoc")
 
-    cmd = ["rsync", "-a", module_input_path, runtime_module_path]
-    subprocess.check_call(cmd)
-    cmd = ["rsync", "-a", swiftdoc_input_path, runtime_swiftdoc_path]
-    subprocess.check_call(cmd)
+    distutils.file_util.copy_file(module_input_path, runtime_module_path,
+            update=1)
+    distutils.file_util.copy_file(swiftdoc_input_path, runtime_swiftdoc_path,
+            update=1)
```

Comment:
Can this use `shutil` instead of `distutils`?

Reply:
The previous semantics would not be preserved in that case.  Using `distutils` means that we can avoid the copy when the files are unchanged.

Reply:
I don't think that is a good enough motivation to require the dependency, though? The files are small and the copies are cheap (as is implementing the mentioned functionality)

Reply:
Sounds good to me then.  I was trying to maintain the semantics of rsync since I figured that there was a reason for that being used rather than copy.  If there isnt, then Im okay with moving this over to shutil.

Reply:
Makes sense... no in this case I don't think its critical, just a copy should be fine